src/components/resource/_fields/custom-coverage/custom-coverage-fields.js

```diff
@@ -134,29 +131,26 @@ const validateWithinPackageRange = (dateRange, packageCoverage, intl) => {
     let packageEndCoverageDate = packageEndCoverage ? moment.utc(packageEndCoverage) : moment.utc();
     let packageRange = moment.range(packageBeginCoverageDate, packageEndCoverageDate);
 
-    let startDate = intl.formatDate(
-      packageBeginCoverageDate,
-      {
-        timeZone: 'UTC',
-        year: 'numeric',
-        month: 'numeric',
-        day: 'numeric'
-      }
-    );
+    let startDate =
+      <FormattedDate
+        value={packageBeginCoverageDate}
+        timeZone="UTC"
```

Comment:
@cherewaty, Shouldn't we adjust the date/time to the local timeZone?

Reply:
I believe in this case, no, since the dates we're dealing with are "fixed" like a user's birthday would be, and do not change based on time zone.